crates/pixi_manifest/src/discovery.rs

```diff
@@ -220,10 +220,11 @@ impl DiscoveryStart {
 
 impl WorkspaceDiscoverer {
     /// Constructs a new instance from the current path.
+    /// It will search the closest package manifest on the way to the workspace.
     pub fn new(start: DiscoveryStart) -> Self {
         Self {
             start,
-            discover_package: false,
+            discover_package: true,
```

Comment:
I think most of the commands we use are not interested in this. I think it would be better to opt-in instead.